### PR TITLE
docs: Convert HISTORY.rst to HISTORY.md

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
All the individual commits in this PR are auto-generated by commands.

**Comment by rht** [comment]
Some of the nested lists are displayed as code blocks instead of nested lists. But they are still readable.

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,1482 @@
+---
+title: Release History
+---
+
+# 2.1.4 (2023-11-7)
+
+This release updates mesa-viz-tornado dependency v0.1.3. This removes
+the external JavaScript templates and
+`prevents 404 errors https://github.com/projectmesa/mesa-viz-tornado/issues/40`\_
```
Comment (tpike3) [inline HISTORY.md:9]: This link set up doesn't seem to convert in MarkDown... needs to be [prevents 404 errors](https://github.com/projectmesa/mesa-viz-tornado/issues/40)

**Comment by tpike3** [comment]
Thanks @rht we need to convert all our .rst to MDs so this is great. I had one comment.

File: HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,1482 @@
...
+-   correct `get_heading` for toroidal space #1686
+
+-   update slider to start at 1FPS #1674
+
+-   update links to examples repo due to creation of mesa_examples
+    #1636, #1637
+
+-
+
+    \*\* CI Improvements\*\*
```
Comment (tpike3) [inline HISTORY.md:203]: Not rendering in bold

**Review by tpike3 — APPROVED** [review]
LGTM

**Pull request merged**